src/server/app.js

```diff
@@ -43,7 +43,7 @@ app.engine('hbs', hbs({
   layoutsDir: path.join(rootDir, 'templates', 'layouts'),
   partialsDir: path.join(rootDir, 'templates', 'partials'),
   helpers: {
-    multiply: (a, b) => (a * b),
+    multiply: (a, b) => (a * b).toFixed(2),
```

Comment:
Optional: It would be best to move formatting code (toFixed) to a separate helper than a logic helper (multiply), or to rename the helper (multiplyAndFormat). Otherwise future user might wonder why they get inaccurate results with "multiply" helper.

Reply:
Good point. This is exactly the kind of thing that causes perplexment ;).

Will change.